pages/dashboard/rewards.tsx

```diff
@@ -0,0 +1,37 @@
+import Head from 'next/head';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import DashboardLayout from 'sections/dashboard/DashboardLayout';
+import RewardsTabs from 'sections/dashboard/Rewards/RewardsTabs';
+import { useAppDispatch, useAppSelector } from 'state/hooks';
+import { fetchClaimableRewardsAll, fetchStakingData } from 'state/staking/actions';
+
+type RewardsComponent = React.FC & { getLayout: (page: HTMLElement) => JSX.Element };
+
+const RewardsPage: RewardsComponent = () => {
+	const { t } = useTranslation();
+	const dispatch = useAppDispatch();
+	const walletAddress = useAppSelector(({ wallet }) => wallet.walletAddress);
```

Comment:
There is a selector `selectWalletAddress` you can use. It's better to try and avoid raw selectors in components so if we ever need to change something it can be done in a single place.

Reply:
That's true. Updated. Thanks for the reminder.